### ISSUE TITLE
ActiveScratchOrg: Child Object: Object does not exist.

### ISSUE THREAD

**Issue by BalassaSamuelson** [afawcett/declarative-lookup-rollup-summaries#648]
I tried to create a rollup from ActiveScratchOrg to User, the relationship name is CreatedBy. However, I get a ` Child Object: Object does not exist.` error.

**Comment by afawcett** [comment]
Try upgrading to the latest version of the tool, this is a new object from API v41 onwards. Let me know how you get on, you must be the first person to use rollups with DX objects, nice! 👍

**Comment by BalassaSamuelson** [comment]
Very nice! Now an email alert is sent to a user if they have 5+ active scratch orgs. Thanks much for the awesome support!

**Comment by BalassaSamuelson** [comment]
Sorry Andrew, must have missed your message. Absolutely, would love to!

**Comment by pgpestoff** [comment]
@afawcett  Hello, I am trying to build lookups with new Health Cloud objects and running into issues with not finding the object via API.

 I tried to create a rollup from CareProgram and CareProgramProducts. Similar to BenSamuels' issue: I get a Child Object: Object does not exist. error. I just installed the declarative lookup summaries in the org last week. Any thoughts on how to update the API version?
